.github/workflows/check-links.yml

```diff
@@ -0,0 +1,49 @@
+name: Check docs links
+on:
+  pull_request:
+    branches: main
+    paths:
+      - docs/**
+
+jobs:
+  check-docs-links:
+    runs-on: ubuntu-latest
+    name: Check links
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Install python
+        uses: actions/setup-python@v2
```

Comment:
This probably isn't needed. Python is already installed: this action is really only if you need a specific version (and it is at v5)

Reply:
Oh interesting. I wonder if I can remove this install for the code styling workflow too

Reply:
Ah nevermind, it's been removed already since we replaced it with the recommended ruff action.